style.css

```diff
@@ -52,3 +52,22 @@ a {
 .is-style-arrow-icon-details[open] > summary {
 	list-style-type: '\2192\00a0\00a0\00a0';
 }
+
+/*
+ * Styles variation for post terms
+ * https://github.com/WordPress/gutenberg/issues/24956
+ */
+
+.is-style-pill a,
+.is-style-pill span:not([class], [data-rich-text-placeholder])
```

Comment:
The prefix and suffix don't need the pill style, that's for when you want to add some text before or after the tag list, so I removed the styling for those

Reply:
Ah awesome - thank you. I'd been trying to figure out how to not have it apply to the placeholder, but couldn't figure it out.

Also - Pill is a much better style name 😂